Documentation: clarify proxmox.yaml key must match Proxmox node name (#5823)
gethomepage/homepage: docs/configs/proxmox.md

---

The Proxmox connection is configured in the `proxmox.yaml` file. See [Create token](#create-token) section below for details on how to generate the required API token.

> You can configure multiple nodes – the key name in the `proxmox.yaml` (e.g. `pve`) must match the `proxmoxNode` field used in your service configuration.

Reviewer: ```suggestion
To configure multiple nodes, ensure the key name in the `proxmox.yaml` matches the `proxmoxNode` field used in your service configuration.
```
